
<h1> Data source </h1> 
<p>
<a
    href="ftp://ftp.arabidopsis.org/home/tair/Genes/TAIR9_genome_release/">athaliana TAIR
    ver 9</a> |
<a
    href="ftp://ftp.jgi-psf.org/pub/JGI_data/phytozome/v5.0/Alyrata">lyrata</a>
|
<a href="ftp://asgpb.mhpcc.hawaii.edu/papaya/">papaya</a> |
<a
    href="ftp://ftp.jgi-psf.org/pub/JGI_data/phytozome/v5.0/Ptrichocarpa/">poplar
ver 2</a>
|
<a href="http://www.genoscope.cns.fr/externe/GenomeBrowser/Vitis/">grape
    ver 2</a><br />
(processed and loaded into <a target="_blank" href="http://genomevolution.org">CoGe</a>)
</p>
<br />

<h1> Methods </h1> 
<p>
For every query arabidopsis gene, we found a 40 gene window centered on
it, and check if we can find syntenic region in the target genome. We used
<a href="http://www.bx.psu.edu/~rsharris/lastz/">LASTZ</a> (default parameters) 
to define anchors. We require syntenic region to have at least 4 collinear anchors 
(out of 40 possible anchors). Nearest anchors on both sides of genes are identified, 
in order to define a "tight" syntenic location. The entire pipeline is a collection of
Python scripts accessible <a target="_blank" 
    href="https://github.com/tanghaibao/positional-history"> here </a>. 
</p>

<p>
Each query gene is then categorized, based on the flank anchors and more sensitive search on the tight
interval. 
</p>
<ul>
    <li> 
    Gene match in the interval - syntelog (<b>S</b>) or no syntelog
    (<b>F</b>)
    </li>
    <li>
    Have both flankers (<b>F</b>) or one flanker (<b>G</b>)
    </li>
    <li>
    No apparent synteny or missing data (<b>-</b>)
    </li>
</ul>

<p>
For genes labeled as <b>F</b>, further validate on two additional checks.
</p>
<ul>
    <li>
    BLAST matches (e.g. to non-coding sequences) in the interval (<b>FB</b>)
    </li>
    <li>
    Assembly gaps (N's) in the interval (<b>FN</b>)
    </li>
</ul>
<p>
Poplar has a lineage-specific genome duplication, so in most cases there are two copies
of orthologous genes. Sometimes the codes are conflicting. For example, one copy
is stable in position (S), another has transposed (F). In this case, our rule is to report the
gene in poplar as stable, i.e. S takes priority over F. The priorities for our
codes are S, F, FB, FN, G, in this order.
</p>
<p>
<a href="/static/example1.png" class="top_up">
<img class="logoimg" src="/static/example1s.png" alt="" / >
</a>
<a href="/static/example2.png" class="top_up">
<img class="logoimg" src="/static/example2s.png" alt="" / >
</a>
</p>
<br />

<h1> Disclaimer </h1>
<p>
This is a whole genome analysis pipeline that is dependent on many rules and
parameters (sometimes arbitrary). We tried our best to minimize the error rate, 
but we note that this data is not completely error-free, although we did manually 
validate about half of the data in this database. Please use with caution.
</p>
<br />
